### DISCUSSION TITLE
Vite instead of webpack?

### DISCUSSION THREAD

**Discussion by ricardo-valero** [necolas/react-native-web#2140] [Q&A] [answered]
Can vite be used instead of webpack?

**Comment by intergalacticspacehighway** [comment]
React native web should work with aliasing in Vite and should not need any additional setup. Can you try this out with aliasing react-native -> react-native-web? https://vueschool.io/articles/vuejs-tutorials/import-aliases-in-vite/

If that works maybe adding it in RNW docs might help. https://necolas.github.io/react-native-web/docs/setup/

**Reply by intergalacticspacehighway** [reply]
Also, you'd need to set the extensions field if you're using RN libraries https://vitejs.dev/config/#resolve-extensions, something like this - https://github.com/vercel/next.js/blob/master/examples/with-react-native-web/next.config.js#L8

**Comment by hxse** [comment, marked as answer]
https://github.com/necolas/react-native-web/discussions/2201#discussioncomment-1992245
